demo/search.html

```diff
@@ -189,24 +184,38 @@ <h1>Symphony Electron Search API Demo</h1>
             if (result.messages.length > 0) {
                 out = result;
                 var th = document.createElement('tr');
+                var avatarTh = document.createElement('td');
+                avatarTh.innerText = "Avatar";
+                var senderNameTh = document.createElement('td');
+                senderNameTh.innerText = "Sender Name";
                 var th1 = document.createElement('td');
                 th1.innerText = "ThreadId";
                 var th2 = document.createElement('td');
                 th2.innerText = 'SenderId';
                 var th3 = document.createElement('td');
                 th3.innerText = 'Text';
+                th.appendChild(avatarTh);
+                th.appendChild(senderNameTh);
                 th.appendChild(th1);
                 th.appendChild(th2);
                 th.appendChild(th3);
                 table.appendChild(th);
                 out.messages.forEach(function (msg) {
                     var tr = document.createElement('tr');
+                    var avatar = document.createElement('img');
+                    avatar.src = msg.senderAvatar;
+                    avatar.style["max-width"] = '75px';
```

Comment:
Always try to use the dot notation for accessing JSON properties or objects unless absolutely necessary.

https://www.w3schools.com/jsref/prop_style_maxheight.asp